bylaws.md

```diff
@@ -88,9 +88,9 @@ c) The corporation adopts and implements some means of verifying (1) that all pe
 
 ### SECTION 8. REGULAR AND ANNUAL MEETINGS ###
 
-Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM, unless such day falls on a legal holiday, in which event the regular meeting shall be held at the same hour and place on the next business day.
+Regular meetings of directors shall be held regularly, at least once a month unless the board votes to not hold a meeting.
```

Comment:
If we change it to quarterly, we can still meet monthly, and better reflect what we do without having to vote on not meeting. These tend to be last minute or in January where we all just want a break. 

```suggestion
Regular meetings of directors shall be held regularly, at least once a quarter, unless the board votes to not hold a meeting.
```

Reply:
> If we change it to quarterly, we can still meet monthly, and better reflect what we do without having to vote on not meeting

How does it get decided when we want to meet or not meet?

Alternatively, it may be easier to go back to "Regular meetings of directors shall be held regularly once a month". If we don't meet quorum, we can't perform any actions anyway and it can't be counted against folks for not attending. The expectation / default should be to meet each month. If people don't show up then the meeting simply doesn't come to order?

Reply:
> How does it get decided when we want to meet or not meet?

The president and/or secretary have historically set the cadence, polled the board, and we set the dates based on availability. If we have two or three directors say that can't make a meeting we have either rescheduled it or canceled it. That was historically because we wouldn't have a quorum so meeting wouldn't matter. 

Big picture: I think the goal here is to better reflect what we do without painting ourselves in a corner. I don't think we need to vote on not having a January meeting because it often falls on a date when most of the board is on vacation.

Reply:
Fair enough, that works for me.